charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```diff
@@ -12,7 +12,78 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              cd /tmp || exit
```

Comment:
```suggestion
              set -o errexit
              set -o nounset
              set -o pipefail

              cd /tmp || exit
```

Reply:
yep

Reply:
for posterity (@coreydaley and I discussed in slack) the `set -o errexit` messed up our repo detection logic  ... but I left nounset and pipefail